components/Enter/EnterPage.tsx

```diff
@@ -14,6 +16,13 @@ const EnterPage = () => {
   const contentRef = useRef()
   const buttonRef = useRef()
 
+  const currentTime = new Date().getTime()
+  const shipDay = "30 Jun 2023 09:00:00 UTC"
```

Comment:
remove `shipDay` and only keep `alertEndDay` 

I will manually ship on Friday 🚢